test/spmd/tensor/test_tp_sharding_ops.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+from spmd.tensor.placement_types import Replicate
+import torch
+from torch.testing._internal.common_utils import run_tests
+from ..test_utils import DistTensorTestBase, with_comms, TEST_GPU_NUM
+from spmd import DeviceMesh, Tensor, Shard, Replicate, distribute_tensor
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+
+
+class TPShardingOpsTest(DistTensorTestBase):
+    # TODO: We need to add CPU tests for ops in the future.
+    @with_comms
+    @skip_if_lt_x_gpu(TEST_GPU_NUM)
```

Comment:
can we try enable this in CPU by removing this and specific cuda logic in the test code? we haven't set up GPU CI yet and it's good for us to test on CPU and be aware in CI that it still works, unless it's like the `dropout` case.

Reply:
I see. Let me remove that.